From ac2a6cfce2ffbd58026f2939fccbc05cd68d3d78 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Fri, 19 Jan 2024 06:46:43 +0100
Subject: [PATCH] com.webos.service.pdm.perm.json: Fix incorrect permission

Fixes:

WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'applicationmanager.launcher' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.pdm.perm.json

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 files/sysbus/com.webos.service.pdm.perm.json.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.pdm.perm.json.in b/files/sysbus/com.webos.service.pdm.perm.json.in
index 411217f..3d5a682 100644
--- a/files/sysbus/com.webos.service.pdm.perm.json.in
+++ b/files/sysbus/com.webos.service.pdm.perm.json.in
@@ -1,7 +1,7 @@
 {
     "com.webos.service.pdm":[
         "account.query",
-        "applicationmanager.launcher",
+        "application.launcher",
         "settings.query",
         "notification.operation",
         "notification.management",
